openmc/mgxs/mdgxs.py

```diff
@@ -55,6 +55,12 @@ class MDGXS(MGXS):
         tallies in OpenMC 'tallies.xml' file.
     delayed_groups : list of int
         Delayed groups to filter out the xs
+    num_polar : Integral, optional
```

Comment:
As for the first, no these particular ones shouldn't match the attributes of Library (or MGXS).  The MGXS and MDGXS have parameter descriptions which are different than the attrib description.